log4cl.asd

```diff
@@ -79,9 +88,8 @@
                              (:file "test-speed")
                              (:file "test-file-category")
                              (:file "test-compat")
-                             (:file "test-regressions")))))
-
-(defmethod perform ((op test-op) (system (eql (find-system :log4cl/test))))
-  (let ((*package* (find-package :log4cl-test)))
-    (eval (read-from-string "(stefil:funcall-test-with-feedback-message 'log4cl-test::test)")))
-  (values))
+                             (:file "test-regressions"))))
+  :perform (test-op (operation coponent)
```

Comment:
ditto, this way of specifying perform methods has been introduced relatively late (for sure after ASDF 2.26).
typo: `s/coponent/component/`